packages/hub/src/app/models/[owner]/[slug]/exports/[variableName]/ModelExportPage.tsx

```diff
@@ -31,6 +32,10 @@ const SquiggleModelExportPage: FC<{
 
   const checkedVersion = useAdjustSquiggleVersion(content.version);
 
+  const [project] = useState(() => {
+    return new SqProject({ linker: squiggleHubLinker });
+  });
```

Comment:
I'm not sure if this is needed. It was used in the /components, for Playground, so I assumed it would be safe to add, but wasn't sure.

Reply:
I don't remember why I went with `useState` (not reactive on linker changes) instead of `useMemo(..., [linker])`, but both are fine for our current goals.